### PR TITLE
Context

### PR DESCRIPTION
**Pull request by SaloniSS**
# Description

Set context to get access to access token and googleID everywhere.

### CODE REVIEW

File: app/App.tsx [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
 import * as Font from 'expo-font';
 import { Ionicons } from '@expo/vector-icons';
 import { NavigationContainer } from '@react-navigation/native';
 import TopStack from './navigation/TopStack';
+import {context} from "./context" ;
 
 const App = () => {
   const [ready, setReady] = useState(false);
+  const [userId, setUserId] = useState(null);
+  const [token, setToken] = useState(null);
```
Comment (ReshmiCode) [inline app/App.tsx:14]: are we going to add tokens everywhere?

File: app/screens/Challenges.tsx [hunk 2]
```diff
@@ -1,7 +1,10 @@
-import React from "react";
+import React, {useContext} from "react";
 import { Container, Content, Text } from "native-base";
+import {context} from "../context" ;
 
 const Challenges = () => {
+  // Get googleID from context._currentValue.googleID
+  console.log("Challenges", context._currentValue.googleID);
```
Comment (ReshmiCode) [inline app/screens/Challenges.tsx:7]: don't you do const auth = useContext(context); then auth.googleID

File: app/components/GoogleLogOut.tsx [hunk 3]
```diff
@@ -1,36 +1,41 @@
...
-            }
-          } catch (e) {
-            console.log("Error with logout", e);
-            return { error: true };
-          }
+      const auth = useContext(context);
+      //reset context
+      auth.logout();
+      try {
+        const { type, accessToken, user } = await Google.logOutAsync({
```
Comment (ReshmiCode) [inline app/components/GoogleLogOut.tsx:19]: should we put this logic inside the logout function

File: app/App.tsx (hunk 1 above)
Comment (SaloniSS) [reply app/App.tsx:14]: No... But if you want to logout, you need the token

File: app/screens/Challenges.tsx (hunk 2 above)
Comment (SaloniSS) [reply app/screens/Challenges.tsx:7]: I tried that.... But it didn’t work.... So I console.log(context) and googleID was nested inside _currentValue

File: app/components/GoogleLogOut.tsx (hunk 3 above)
Comment (SaloniSS) [reply app/components/GoogleLogOut.tsx:19]: This is the logout function?

**Pull request merged**